bindings/nodejs/lib/client/client-method-handler.ts

```diff
@@ -22,14 +22,15 @@ export class ClientMethodHandler {
         }
     }
 
-    async destroy(): Promise<void> {
-        return destroyClient(this.methodHandler);
+    async destroy() {
+        await destroyClient(this.methodHandler);
+        this.methodHandler = undefined;
     }
 
     async callMethod(method: __ClientMethods__): Promise<string> {
         return callClientMethodAsync(
             JSON.stringify(method),
-            this.methodHandler,
+            this.methodHandler!,
```

Comment:
Forbidden non-null assertion

Reply:
i know, see my question above ^
I can change them all to ? i suppose, but not sure

Reply:
Ah wasn't sure about what you mean with the question, but I think it's not useful, at least haven't seen anywhere that this should help